src/include/cdb/cdbvars.h

```diff
@@ -309,6 +309,8 @@ typedef enum GpVars_Interconnect_Type
 	INTERCONNECT_TYPE_TCP = 0,
 	INTERCONNECT_TYPE_UDPIFC,
 	INTERCONNECT_TYPE_PROXY,
+	INTERCONNECT_TYPE_UDP2,
+	INTERCONNECT_TYPE_NUM, /* it must be the last one */
```

Comment:
I don't recommend define the enum value to mean the max number of IC, at least the name pattern should not be the same.

Reply:
fix it with [commit](https://github.com/cloudberrydb/cloudberrydb/pull/646/commits/cc3a12c740074a515a04896ea27a63cdf46ace57)